setup.py

```diff
@@ -65,6 +66,7 @@ def get_version(module='pylsp'):
         'pylint': ['pylint>=2.5.0'],
         'rope': ['rope>0.10.5'],
         'yapf': ['yapf'],
+        'autobahn-twisted': ['autobahn[twisted,accelerate]>=21.11.1'],
```

Comment:
I wonder if the extras should be renamed to `websockets` so it is not tied to the current implementation if we need to change it in the future.